src/UserSettingsStore.js

```diff
@@ -180,33 +190,30 @@ export default {
         localStorage.setItem('mx_local_settings', JSON.stringify(settings));
     },
 
-    getFeatureById(feature: string) {
-        for (let i = 0; i < this.LABS_FEATURES.length; i++) {
-            const f = this.LABS_FEATURES[i];
-            if (f.id === feature) {
-                return f;
-            }
-        }
-        return null;
-    },
-
     isFeatureEnabled: function(featureId: string): boolean {
-        // Disable labs for guests.
-        if (MatrixClientPeg.get().isGuest()) return false;
+        const featuresConfig = SdkConfig.get()['features'];
 
-        const feature = this.getFeatureById(featureId);
-        if (!feature) {
-            console.warn(`Unknown feature "${featureId}"`);
-            return false;
+        let sdkConfigValue = 'labs';
+        if (featuresConfig && featuresConfig[featureId] !== undefined) {
+            sdkConfigValue = featuresConfig[featureId];
         }
-        // Return the default if this feature has an override to be the default value or
-        // if the feature has never been toggled and is therefore not in localStorage
-        if (Object.keys(feature).includes('override') ||
-            localStorage.getItem(`mx_labs_feature_${featureId}`) === null
-        ) {
-            return feature.default;
+
+        if (sdkConfigValue === 'enable') {
+            return true;
+        } else if (sdkConfigValue === 'enable') {
```

Comment:
Probably need a negation of some kind here :wink:

Reply:
oops